### ISSUE TITLE
Tab containers and tabs don't follow theme colour schemes

### ISSUE THREAD

**Issue by sk82jack** [ironmansoftware/universal-dashboard#1006]
I'm using the Azure theme but when I add a tab container and some tabs they aren't following the colour scheme:

![image](https://user-images.githubusercontent.com/20758758/62431010-9139e000-b71b-11e9-985a-b5cb392c67b6.png)

I am running on PS 7.0.0-preview.2 and UD.Community v2.5.2

**Comment by psDevUK** [comment]
Hi @sk82jack  you could set the colour scheme for the tabs as a theme to get around this issue:-
    '.tabs .tab'                                                                                                                    = @{
        'color' = "#000"
    }

    '.tabs .tab a:hover'                                                                                                            = @{
        'background-color' = "#033584"
        'color'            = "#ffffff"
    }

    '.tabs .tab a.active'                                                                                                           = @{
        'background-color' = "#4392f1"
        'color'            = "#ffffff"
    }
    '.tabs .tab a:focus.active'                                                                                                     = @{
   'background-color' = "#4392f1"
        'color'            = "#ffffff"
    }
    '.tabs .indicator'                                                                                                              = @{
        'background-color' = "#339933"
    }
    '.tabs .tab a'                                                                                                                  = @{
        'color' = "#000000"
    }

**Comment by sk82jack** [comment]
Thanks @psDevUK I will have a play around with this later 🙂

**Comment by psDevUK** [comment]
No worries @sk82jack I went with my own blue theme for a dashboard.  The complete file can be found here:- https://github.com/psDevUK/psUniversalDashboard/blob/master/PagesGitHub/Theme.ps1

**Comment by adamdriscoll** [comment]
This issue now has a bounty: https://www.bountysource.com/issues/78224077-tab-containers-and-tabs-don-t-follow-theme-colour-schemes

**Comment by psDevUK** [comment]
I got 4 kids to feed so I posted my answer here https://github.com/psDevUK/UDscripts/blob/master/AzureTheme.ps1
 It's all about feeding your kids right? lol

**Comment by adamdriscoll** [comment]
@psDevUK  Can you put this into a PR that we can merge?

**Comment by sk82jack** [comment]
@psDevUK that theme still has a white sidenav bar and white collapsible items. The other thing I noticed about the Azure theme is the text on the grid filters is black and very hard to read:
![image](https://user-images.githubusercontent.com/20758758/62901831-a46d3100-bd55-11e9-9070-ba072f57ebce.png)

Love the blue on the tabs though 😍 I'll see if I can play around with some other settings and make something PR worthy but I'm pretty new to UD and have no experience with HTML/CSS so we'll see 😅

**Comment by psDevUK** [comment]
@sk82jack yep I can fix the collapsible and the side-nav and the pagenation will look at this later today to finish

**Comment by sk82jack** [comment]
I was having a play around last night and added the following settings to what you had already which I thought looked pretty good (one I wasn't really sure about was modal background so just chose a lighter grey). I was gonna see if I can add some theme mappings for some of these to make it easier for people going forward but it was getting pretty late last night so will take a look this evening.

```ps

UDCollapsible = @{
    BackgroundColor = '#252525'
}
UDCollapsibleItem = @{
    BackgroundColor = '#252525'
    FontColor       = '#FFFFFF'
}
'input:not([type]), input[type="date"]:not(.browser-default), input[type="datetime-local"]:not(.browser-default), input[type="datetime"]:not(.browser-default), input[type="email"]:not(.browser-default), input[type="number"]:not(.browser-default), input[type="password"]:not(.browser-default), input[type="search"]:not(.browser-default), input[type="tel"]:not(.browser-default), input[type="text"]:not(.browser-default), input[type="time"]:not(.browser-default), input[type="url"]:not(.browser-default), textarea.materialize-textarea' = @{
    'color' = '#FFFFFF'
}
'.pagination li.active' = @{
    'background-color' = '#55b3ff'
}
'.pagination li.active a' = @{
    'color' = '#444'
}
'.pagination a:hover:not(.active)' = @{
    'background-color' = '#55b3ff'
    'border-radius'    = '2px'
}
'.sidenav' = @{
    'background-color' = '#252525'
}
'.sidenav .collapsible-body, .sidenav.fixed .collapsible-body' = @{
    'background-color' = '#252525'
}
'.sidenav li > a' = @{
    'color' = '#FFFFFF'
}
'.sidenav .subheader' = @{
    'color' = '#FFFFFF'
}
'.card .card-action a:not(.btn):not(.btn-large):not(.btn-small):not(.btn-large):not(.btn-floating)' = @{
    'color' = '#55b3ff'
}
'.modal' = @{
    'background-color' = '#404040'
}
```

**Comment by psDevUK** [comment]
Hi @sk82jack and @adamdriscoll  I never done a pull request before but I now believe I have set one up and it is here:- https://github.com/psDevUK/UDscripts/pull/2/commits/d812df2bfdfae7140f4b98d0b25e7f49ee5ee87f
I have done a lot more styling on this theme now.

**Comment by sk82jack** [comment]
@psDevUK cool thanks - I'll take a look and report back with my feedback 🙂

It looks like you've PR'd against your own repo though so what you'll need to do is fork the universal-dashboard repo and then edit the file at https://github.com/ironmansoftware/universal-dashboard/blob/master/src/UniversalDashboard/Themes/Azure.psd1 (but the one in your forked repo) and then make a pull request from your forked repo against the main UD repo.

**Comment by psDevUK** [comment]
ok cool beans think I am on the same level now? I have attempt 2 located here:-
https://github.com/psDevUK/universal-dashboard/pull/1
I did try to remember all the things I changed and typed them out...hoping you like what you see 👍

**Comment by sk82jack** [comment]
@psDevUK closer but you've PR'd against your own fork's master branch instead of the this repo 😅 When you go to your fork and create the pull request make sure it's the ironmansoftware/universal-dashboard-master branch on the left (I assume you want PRs targetted at master based on previous PRs @adamdriscoll but let me know if that's not correct) and it's your fork-branch on the right like so

![image](https://user-images.githubusercontent.com/20758758/62977398-b5ca4200-be16-11e9-8806-b7c82ec5657d.png)

**Comment by psDevUK** [comment]
Thanks @sk82jack like they say third time lucky...ok so I really do now believe I done this correctly with your help.  Hopefully you like what was done, anything else you want modifying let me know.

**Comment by sk82jack** [comment]
@psDevUK Great work - looks awesome! I do have some more edge cases which don't appear to be covered in what you've done so far:

Sidenav expandable sections and subheaders are hard to read:
![image](https://user-images.githubusercontent.com/20758758/62978861-30489100-be1a-11e9-8c3b-434b6737e0fd.png)

Also hovering over links in the sidenav makes them orange instead of blue.

Would be nice to have blue links on cards instead of orange:
![image](https://user-images.githubusercontent.com/20758758/62979690-638c1f80-be1c-11e9-922e-d554da2297de.png)

The highlighted grid page is red:
![image](https://user-images.githubusercontent.com/20758758/62980443-84557480-be1e-11e9-831d-ad87bac9c9fb.png)

Collapsibles have a white bottom border when expanded:
![image](https://user-images.githubusercontent.com/20758758/62983604-e23a8a00-be27-11e9-9c66-233692eb73b3.png)

**Comment by psDevUK** [comment]
Thanks @sk82jack, I will look into what else you discovered was not being covered by the theme I modified. Hoping I will get these last few issues corrected today/tonight

**Comment by psDevUK** [comment]
Hey @sk82jack  I know this issue is now closed, but I don't consider my-self as a half-job so I have now made the additional modifications, if there is anything else I missed let me know but all the above requests now fixed...I also like using dividers between my menu items so I pimped that item too. Anyways was real good to do this and help out. Was wondering @adamdriscoll you up for me doing the same to the other themes?
